lib/lotus/root.rb

```diff
@@ -0,0 +1,7 @@
+require 'pathname'
+
+module Lotus
+  def self.root
+    Lotus::Configuration.new.root
```

Comment:
IMO, we shouldn't create a new instance of configuration, we have to take the current cofiguration